lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,27 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let kTypeStrippingMode = null;
+ObjectDefineProperty(process.features, 'typescript', {
```

Comment:
My only nit would be that, since we are exposing different modes of TS, that the property could be called `typescript_mode`, this would be more concise with `strip/transform` and `false`.

However, I do not have a strong instance about this, so feel free to ignore if you think it doesn't make sense.

Reply:
Oh I see @legendecas already commented that

Reply:
Yep, I didn't see these reviews. I'll fix up and prepare this PR later today.